WcaOnRails/app/models/country.rb

```diff
@@ -6,7 +6,7 @@ class Country < ApplicationRecord
   belongs_to :continent, foreign_key: :continentId
   has_many :competitions, foreign_key: :countryId
 
-  scope :uncached_real, -> { where("name not like 'Multiple Countries%'") }
+  scope :uncached_real, -> { where.not(id: %w(XA XE XS)) }
```

Comment:
Didn't notice we had a similar test here!
Then I think a frozen static array (that we would use in both places) would be better: just because if we need to change this array for some reason, then we wouldn't forget about changing it else where!